htmlbook-autogen/document.html.erb

```diff
@@ -7,7 +7,8 @@ xmlns="http://www.w3.org/1999/xhtml">
 <% toc_levels = (attr? :levels) ? (attr :levels).to_i : (@document.attr :toclevels, 2).to_i %>
   <head>
     <title><%= doctitle %></title>
-    <meta name="<%= doctitle %>" content="text/html; charset=utf-8"/>
+    <meta charset="utf-8"/>
+    <meta name="title" content="<%= doctitle %>"/>
```

Comment:
An interesting question. I was wavering between the two options before deciding on the newer one, mainly because we are generating HTML5. I am happy to change to the more traditional approach. Regarding EPUB, looking at the EPUB 2.0.1 specification, it says that conformant XHTML content must be either UTF-8 or UTF-16 so I am not sure whether an EPUB reader is even going to consider a meta element specifying the encoding. That said, I am happy to make the change you suggest.